### PR TITLE
Shows selected color ramp 

### PR DESCRIPTION
**Pull request by javierarce**
Shows selected ramp state in the ramp selector.

This PR also replaces the `.CDB-ColorBar` with `.ColorBar`.

animation

### CODE REVIEW

**Comment by javierarce** [comment]
CR: @xavijam 
CC: @piensaenpixel

File: app/assets/stylesheets/editor-3/color-bar.css.scss [hunk 1]
```diff
@@ -0,0 +1,107 @@
+@import 'cdb-utilities/mixins';
+@import 'cdb-variables/colors';
+@import 'cdb-variables/sizes';
+
+
+.CDB-ListDecoration-itemLink.is-selected .ColorBarContainer {
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/color-bar.css.scss]: Those CDB whatever, are necessary? Could we move them to the editor stylesheets?

**Comment by xavijam** [comment]
Could you check CSS changes @piensaenpixel?
About js changes 👍

**Comment by javierarce** [comment]
@xavijam: @piensaenpixel did the changes, so I think he likes them :)

File: app/assets/stylesheets/editor-3/color-bar.css.scss (hunk 1 above)
Comment (javierarce) [inline app/assets/stylesheets/editor-3/color-bar.css.scss]: Mmm, what do you mean? The first two are styling the `..ColorBarContainer` of the file.

Comment (xavijam) [inline app/assets/stylesheets/editor-3/color-bar.css.scss]: I wonder if CDB-ListDecoration is something that lives in CartoAssets or we could move it to the Editor. But in any case, let's tackle it in other issue. 👍

**Comment by xavijam** [comment]
👍

File: app/assets/stylesheets/editor-3/color-bar.css.scss (hunk 1 above)
Comment (javierarce) [inline app/assets/stylesheets/editor-3/color-bar.css.scss]: Ok!

**Pull request merged**